### PR TITLE
[READY FOR REVIEW] Added support table with visualizations and search table

### PR DESCRIPTION
**Pull request by KjellKod**
screen shot 2018-03-23 at 12 51 56 am
screen shot 2018-03-23 at 12 54 39 am

### CODE REVIEW

**Comment by Schatzman** [comment]
@KjellKod all of the visualizations work and are visualizing the correct things.

It strikes me that both of these dashboards appear very empty. The visualizations are small and when I have the page fullscreen, there is a lot of empty space at the bottom of both of these dashboards. Could you consider making the visualizations a bit larger so there's not so much empty space?

screen shot 2018-04-06 at 10 31 23 am
screen shot 2018-04-06 at 10 31 16 am

**Comment by craig-cogdill** [comment]
I think we need Liam to weigh in on the naming of the dashboards as well. We might want to change 'Support' to 'Diagnostic' or something.

File: resources/visualizations/Top-Applications-By-Metadata-Impact-(pie).json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+   "title":"Top Applications By Metadata impact (pie)",
```
[the remark this answers is not in the document]
Comment (KjellKod) [reply resources/visualizations/Top-Applications-By-Metadata-Impact-(pie).json]: thanks for noticing. addressed.

File: resources/dashboards/Support-Dashboard-Where.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+   "title":"Support Dashbboard - Where",
```
[the remark this answers is not in the document]
Comment (KjellKod) [reply resources/dashboards/Support-Dashboard-Where.json]: updated

**Comment by KjellKod** [comment]
@craig-cogdill updated the bugs.   I'm leaving the size and naming of the dashboards as is and will let Liam weigh in on it.

**Comment by lm77** [comment]
In general, we should rename all the dashboard to remove the "Dashboard" at the end, but we'll do that at a later time.

Diagnostics is a good term for this. It will cause short-term confusion with our Diagnostics menu, but that is proposed to be removed and replaced with LRMetrics. 

How about:
* "Diagnostics: Applications Dashboard"
* "Diagnostics: IP Dashboard"

I agree that the initial impression of the design is a bit sparse. How about we change them as follows:
* Add a text box to the top of both with instructions about these special dashboards. We can work on the text separately. Something along the lines of "The Diagnostics: Applications Dashboard is when investigating the performance of a NetMon system. Use this dashboard to determine [...]. It's recommended that you customize this dashboard to suit your specific use cases. For more information, please see [NetMon Help]."
* Move the four small charts horizontally [][][][] across the top and under the new text box.

**Comment by KjellKod** [comment]
User story for taking this from beta to version 1.0: https://jira.logrhythm.com/browse/NM-1049

**Pull request merged**